src-tauri/src/aux_data_structures/cut.rs

```diff
@@ -0,0 +1,23 @@
+pub struct Cut {
+    weight: f64,
+    a: String,
+    b: String,
```

Comment:
Would it be worth making this generic? Idk what other values of `a` or `b` could be, or are they always going to be strings?

Reply:
Yeah a and b are the names of the nodes. They are the roots of their respective disjoint subsets.